flexmeasures/data/models/generic_assets.py

```diff
@@ -281,6 +295,22 @@ def get_inflexible_device_sensors(self):
         Searches for inflexible_device_sensors upwards on the asset tree
         This search will stop once any sensors are found (will not aggregate towards the top of the tree)
         """
+
+        from flexmeasures.data.models.time_series import Sensor
+
+        if not self.inflexible_device_sensors:
```

Comment:
Maybe a distinction between `None` and `[]` would be useful? I'm not sure if this is ever `None`, but then if it's `[]`m we might not need this query?

Reply:
Unfortunately it turns out inflexible_device_sensors is never None, but equals to [] until the query was done